shipment.go

```diff
@@ -119,6 +119,10 @@ type getShipmentRatesResponse struct {
 	Rates *[]*Rate `json:"rates,omitempty"`
 }
 
+type getShipmentStatelessRatesResponse struct {
```

Comment:
Need to fix the references too

```suggestion
type getStatelessRatesResponse struct {
```